fractions/opentracing/src/main/resources/modules/io/opentracing/main/module.xml

```diff
@@ -0,0 +1,14 @@
+<module xmlns="urn:jboss:module:1.3" name="io.opentracing">
+  <resources>
```

Comment:
Are all these APIs only? ie. the user still needs to bring an impl in their project dependencies?